synctoolui/src/main/java/org/duracloud/syncui/SyncUIDriver.java

```diff
@@ -85,16 +88,34 @@ private static void launchServer(final String url,
             contextPath = SyncUIConfig.getContextPath();
             Server srv = new Server(port);
 
+            // Setup JMX
+            MBeanContainer mbContainer = new MBeanContainer(
+                    ManagementFactory.getPlatformMBeanServer() );
+            srv.addBean( mbContainer );
+
             ProtectionDomain protectionDomain =
                 org.duracloud.syncui.SyncUIDriver.class.getProtectionDomain();
             String warFile =
                 protectionDomain.getCodeSource().getLocation().toExternalForm();
+
             log.debug("warfile: {}", warFile);
-            final WebAppContext context = new WebAppContext();
+            WebAppContext context = new WebAppContext();
             context.setContextPath(contextPath);
-            context.setAttribute("extractWAR", Boolean.FALSE);
             context.setWar(warFile);
+            context.setExtractWAR(Boolean.TRUE);
+
+            Configuration.ClassList classlist = Configuration.ClassList
+                    .setServerDefault( srv );
+            classlist.addBefore(
+                    "org.eclipse.jetty.webapp.JettyWebXmlConfiguration",
+                    "org.eclipse.jetty.annotations.AnnotationConfiguration" );
+

```

Comment:
@nwoodward :  another question:  is this line just boiler plate or is this something you inserted to get it to work.  Either way a comment mentioning its purpose for future reference would be helpful.